GroupBundle/EventSubscriber/NodeGroupRoleTransformSubscriber.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace OpenOrchestra\GroupBundle\EventSubscriber;
+
+use OpenOrchestra\ApiBundle\Transformer\NodeGroupRoleTransformer;
+use OpenOrchestra\ApiBundle\Transformer\TransformerWithGroupInterface;
+use OpenOrchestra\GroupBundle\Event\GroupFacadeEvent;
+use OpenOrchestra\GroupBundle\GroupFacadeEvents;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelInterface\Model\ReadSiteInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use UnexpectedValueException;
+
+/**
+ * Class NodeGroupRoleTransformSubscriber
+ */
+class NodeGroupRoleTransformSubscriber implements EventSubscriberInterface
+{
+    protected $router;
+    protected $transformer;
+
+    /**
+     * @param UrlGeneratorInterface $router
+     */
+    public function __construct(UrlGeneratorInterface $router, NodeGroupRoleTransformer $transformer)
+    {
+        $this->router = $router;
+        $this->transformer = $transformer;
+    }
+
+    /**
+     * @param GroupFacadeEvent $event
+     */
+    public function postGroupTransformation(GroupFacadeEvent $event)
```

Comment:
A method name should always be a verb, so maybe more postGroupTransform in this case